core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -442,17 +474,24 @@ var StoryEditorPage = function() {
     await waitFor.elementToBeClickable(
       addPrerequisiteSkillButton,
       'addPrerequisitesSkillButton takes too long to be clickable');
-    await addPrerequisiteSkillButton.click();
+    await action.click(
+      'Add Prerequisite Skill Button', addPrerequisiteSkillButton);
     var selectSkillModal = await this.getSelectSkillModal();
     await selectSkillModal.selectSkill(skillName);
   };
 
   this.deleteAcquiredSkillByIndex = async function(index) {
-    await deleteAcquiredSkillButton.get(index).click();
+    await action.click(
+      'Delete Acquired Skill Button',
```

Comment:
Can you include `index` in this message so we know which button is failing to load?